src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -25,23 +25,6 @@ public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
         AddSentryToXcodeProject(options, cliOptions, logger, pathToProject);
     }
 
-    internal static bool IsNativeSupportEnabled(SentryUnityOptions options, IDiagnosticLogger logger)
```

Comment:
We can remove this because we no longer know for sure whether the native SDK won't be enabled at runtime.